prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -46,6 +67,7 @@ data class Hl7Configuration
     val messageProfileId: String?,
     val reportingFacilityName: String? = null,
     val reportingFacilityId: String? = null,
+    val reportingFacilityIdType: String? = null,
```

Comment:
Instead of a CLIA, WA wants an OID that they specify. This allows the default of CLIA to overridden with ISO.

Reply:
Does this change our thinking about requiring a CLIA field in the qualityFilter?